app.py

```diff
@@ -14,8 +14,15 @@
 import sys
 
 from app import app
-sys.path.insert(0,'/home/ubuntu/workspace/')
-# sys.path.insert(0,'/Users/jesson/Development/Github/Project_ART')
+
+if os.getenv("LOCAL"):
+    if os.getenv("LOCAL").lower() == 'true':
+        sys.path.insert(0,os.getcwd())
+    else:
+        sys.path.insert(0,os.getenv("LOCAL"))
+else:
+    sys.path.insert(0,'/home/ubuntu/workspace/')
```

Comment:
If `sys.path.insert(0,os.getcwd())` works locally shouldn't it also work for the c9 enviroment?

Reply:
Yes it should.
I will remove the else

Reply:
Oh that line was kept, so that it defaults to the c9 path even when no LOCAL option is passed to python

Reply:
Also using only `sys.path.insert(0,os.getcwd())` might cause it to fail on an actual server, if the server access it from outside of the git root folder, although I am not sure on that